lib/Models/Catalog/Ows/OwsInterfaces.ts

```diff
@@ -6,6 +6,85 @@ export interface OnlineResource {
+  readonly rangeClosure?: RangeClosureType;
+
+  /** Maximum value of this numeric parameter. */
+  readonly MaximumValue?: string;
+  /** Minimum value of this numeric parameter. */
+  readonly MinimumValue?: string;
+  /** The regular distance or spacing between the allowed values in a range. */
+  readonly Spacing?: string;
+}
+
+interface AllowedValuesType {
+  readonly Range?: RangeType | RangeType[];
+  readonly Value?: string | string[];
+}
+
+export interface DomainType {
+  /** Name or identifier of this quantity. */
+  name: string;
+  /** List of all the valid values and/or ranges of values for this quantity. For numeric quantities, signed values should be ordered from negative infinity to positive infinity. */
+  readonly AllowedValues: AllowedValuesType;
+}
+
+export interface RequestMethodType extends OnlineResource {
+  /** Optional unordered list of valid domain constraints on non-parameter quantities that each apply to this request method for this operation. If one of these Constraint elements has the same "name" attribute as a Constraint element in the OperationsMetadata or Operation element, this Constraint element shall override the other one for this operation. The list of required and optional constraints for this request method for this operation shall be specified in the Implementation Specification for this service. */
+  readonly Constraint?: DomainType | DomainType[];
+}
+
+interface HTTPType {
+  /** Connect point URL prefix and any constraints for the HTTP "Get" request method for this operation request. */
+  readonly Get: RequestMethodType | RequestMethodType[];
+  /** Connect point URL and any constraints for the HTTP "Post" request method for this operation request. */
+  readonly Post: RequestMethodType | RequestMethodType[];
+}
+
+interface DCPType {
+  /** Connect point URLs for the HTTP Distributed Computing Platform (DCP). Normally, only one Get and/or one Post is included in this element. More than one Get and/or Post is allowed to support including alternative URLs for uses such as load balancing or backup. */
+  readonly HTTP: HTTPType;
+}
+
+interface OperationType {
+  /** Name or identifier of this operation (request) (for example, GetCapabilities). The list of required and optional operations implemented shall be specified in the Implementation Specification for this service. */
+  readonly name: string;
+
+  /** Optional unordered list of valid domain constraints on non-parameter quantities that each apply to this operation. If one of these Constraint elements has the same "name" attribute as a Constraint element in the OperationsMetadata element, this Constraint element shall override the other one for this operation. The list of required and optional constraints for this operation shall be specified in the Implementation Specification for this service. */
+  readonly Constraint?: DomainType | DomainType[];
```

Comment:
The JSDoc describes this member as a "list of valid domain constraintS", but the member is  named `Constraint` (without the S). Would it make sense to rename the members that are in lists to have the plural s at the end of their names so it's clear from the name that it's (potentially) multiple things being dealt with?

Reply:
The property name and JSdoc are taken from the standard and XSD schema, so we can't change it